vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_demos/config_files/aws_iot_network_config.h

```diff
@@ -51,7 +51,10 @@
  * in "aws_iot_network.h"
  *
  */
-
+#ifndef M487_ETH_DEMO
 #define configENABLED_NETWORKS      ( AWSIOT_NETWORK_TYPE_WIFI )
+#else
+#define configENABLED_NETWORKS      ( AWSIOT_NETWORK_TYPE_ETH )
+#endif
```

Comment:
So we don't want to enable both at the same time. Is there some problem with that?

Reply:
@qiutongs   Yes, SRAM 96KB resource can't afford to enable two connectivity concurrently in this aws_demo.

Reply:
ok